packs/st2/actions/lib/action.py

```diff
@@ -20,21 +20,24 @@ def __init__(self, config):
         self.client = self._get_client()
 
     def _get_client(self):
-        base_url, api_url = self._get_st2_urls()
+        base_url, api_url, auth_url = self._get_st2_urls()
         token = self._get_auth_token()
-        return self._client(base_url=base_url, api_url=api_url, token=token)
+        return self._client(base_url=base_url, api_url=api_url,
+                            auth_url=auth_url, token=token)
 
     def _get_st2_urls(self):
         # First try to use base_url from config.
         base_url = self.config.get('base_url', None)
-        api_url = None
+        api_url = self.config.get('api_url', None)
+        auth_url = self.config.get('auth_url', None)
 
         # not found look up from env vars. Assuming the pack is
         # configuered to work with current StackStorm instance.
         if not base_url:
             api_url = os.environ.get('ST2_ACTION_API_URL', None)
+            auth_url = os.environ.get('ST2_ACTION_AUTH_URL', None)
```

Comment:
I just noticed we don't actually set this variable in st2 for actions.

Not a big deal, but something to look into.